tests/tracer/test_encoders.py

```diff
@@ -51,6 +51,7 @@ def gen_trace(nspans=1000, ntags=50, key_size=15, value_size=20, nmetrics=10):
 
             if not root:
                 root = span
+                root.set_tag("_dd.origin", dd_origin)
```

Comment:
Does this set the root span's context `_dd.origin`?

Reply:
Also, does this mean that `dd_origin` is only set on the root span, or is it also propagated to every span in the trace as well?

Reply:
oh, no it doesn't good catch, this should be `root.context.dd_origin = dd_origin` 👍🏻